packages/react-docs-example/ts/docs.tsx

```diff
@@ -57,8 +57,8 @@ const docsInfoConfig: DocsInfoConfig = {
                 'https://github.com/0xProject/web3-typescript-typings/blob/f5bcb96/index.d.ts#L314',
             LogEntry: 'https://github.com/0xProject/web3-typescript-typings/blob/f5bcb96/index.d.ts#L366',
             FilterObject: 'https://github.com/0xProject/web3-typescript-typings/blob/f5bcb96/index.d.ts#L109',
-            ['Web3.BlockParam']: 'https://github.com/0xProject/web3-typescript-typings/blob/f5bcb96/index.d.ts#L278',
-            ['Web3.ContractAbi']: 'https://github.com/0xProject/web3-typescript-typings/blob/f5bcb96/index.d.ts#L47',
+            ['BlockParam']: 'https://github.com/0xProject/web3-typescript-typings/blob/f5bcb96/index.d.ts#L278',
+            ['ContractAbi']: 'https://github.com/0xProject/web3-typescript-typings/blob/f5bcb96/index.d.ts#L47',
```

Comment:
If these are no longer using `web3-typescript-typings`, remove them entirely from this list and add them under `publicTypes` above.